src/Element/BaseInput.php

```
@@ -99,8 +119,10 @@ protected function buildElement(string|Stringable $content = '', array $tokenVal
         $id = $this->getAttribute('id', null);
         $ariaDescribedBy = $this->getAttribute('aria-describedby', null);
 
+        $ariaDescribedBySuffix = $this->ariaDescribedBySuffix === '' ? '-help' : $this->ariaDescribedBySuffix;
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $ariaDescribedBySuffix. Keep variable name length under 20. (undefined)

(LongVariable)